### ISSUE TITLE
FCMDevice.send_message() method: bug in params

### ISSUE THREAD

**Issue by vadim-bulochnik** [xtrinch/fcm-django#4]
Hello,

There is a bug for sure in FCMDevice.send_message() method: you always provide "None" for "sound" and "badge" params:

```
result = fcm_send_message(
			registration_id=self.registration_id,
			title=title,
			body=body,
			icon=icon,
			data=data,
			sound=None,
			badge=None,
			**kwargs
		)
```
As a result - no sound on iOS.

Best regards,
Vadim

**Comment by xtrinch** [comment]
Thanks, use kwargs for now, until I update the package :)

**Comment by xtrinch** [comment]
Fixed in release 0.2.5
